bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java

```diff
@@ -150,6 +152,13 @@ public KeyValueStorageRocksDB(String path, DbConfigType dbConfigType, ServerConf
                 options.setTableFormatConfig(tableOptions);
             }
 
+            // Configure file path
+            String logPath = conf.getString(ROCKSDB_LOG_PATH, "");
+            if (!logPath.isEmpty()) {
+                options.setDbLogDir(FileSystems.getDefault().getPath(logPath, subPath).toFile().toString());
+            }
+            String path = FileSystems.getDefault().getPath(basePath, subPath).toFile().toString();
```

Comment:
Can we log this path?

Reply:
I made the changes requested. I used commit --amend, so you won't see a second commit with the fix.